x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -263,6 +259,7 @@ export interface PendingActionsResponse {
 
 export type PendingActionsRequestQuery = TypeOf<typeof ActionStatusRequestSchema.query>;
 
+export type ResponseActionStatus = typeof RESPONSE_ACTION_STATUS[number];
```

Comment:
FYI: if you wanted to avoid deriving the type here from `RESPONSE_ACTION_STATUS`, you could just add the type to the `constants` module and export it out of there.

Reply:
Yeah that would be better. Will do

Reply:
524c760b6a3cc8ab4728a85c77cbfca85e2bad45